README.md

```diff
@@ -13,7 +13,7 @@ Zen, your in-app firewall for peace of mind– at runtime.
 
 Zen by Aikido is an embedded Web Application Firewall that autonomously protects Node.js apps against common and critical attacks.
 
-It protects your Node.js apps by preventing user input containing dangerous strings, which allow injection, pollution, and path traversal attacks. It runs on the same server as your Node.js app for simple [installation](#installation) and zero maintenance.
```

Comment:
This sentence is still used in the other repos (e.g. python).

Reply:
Indeed, once this merged will change and streamline the README.md, because e.g. I don't think the LICENSE section is same everywhere